ElectronClient/app/app.js

```diff
@@ -1037,6 +1091,13 @@ class Application extends BaseApplication {
 			css: cssString
 		});
 
+		const templates = await TemplateUtils.loadTemplates(Setting.value('profileDir') + '/templates');
```

Comment:
Could you create a "templateDir" constant for this (using `Setting.setConstant(...`)?

Reply:
Done!